src/icp_canister_backend/tests/test.rs

```diff
@@ -1171,7 +1181,14 @@ fn test_slash_function() {
     );
 
     // Test withdrawal after slash - advance time first
-    advance_time(&pic, EPISODE_DURATION + 1);
+    // Advance time to ensure stakable episodes have expired
+    let current_episode = get_current_episode(&pic, canister_id, user);
+    let episodes_to_advance = if first_stakable_episode >= current_episode {
```

Comment:
I believe it is possible to calculate easier like:
episodes_to_advance = first_stakable_episode - current_episode + 1
ALSO
you can make a function that takes an episode number and returns time to advance.